frontend/cypress/tests/chartView/chartView_toolBar_searchBox.cy.js

```diff
@@ -33,4 +33,53 @@ describe('search bar', () => {
       expect(children).to.equal(1);
     });
   });
+
```

Comment:
We could add more test cases:
* if the search term does not start with the 'tag:' prefix but still has it: 'tag-1 tag:tag-2'
* if there is no 'tag:' prefix in the search term at all: 'tag-1 tag-2'
* if there are multiple tags after the prefix: 'tag: tag-1 tag-2'

These tests may be considered too much for end-to-end testing, but since we do not have unit tests currently, I think they should be added.

Reply:
just added the tests!